packages/lib/src/vis/tiles/TiledHeatmapMesh.tsx

```diff
@@ -74,25 +76,32 @@ function TiledHeatmapMesh(props: Props) {
 
   const { colorMap, invertColorMap = false } = colorMapProps;
 
+  const tooltipLayer = layers[layers.length - 1];
```

Comment:
Shouldn't the `tooltipLayer` be the `currentLayerIndex`?

Reply:
In essence, this is `currentLayerIndex` but you are right. It would be better to be explicit.